api-testing/multicloud-api/src/main/java/com/soda/utils/ConstantUrl.java

```diff
@@ -0,0 +1,61 @@
+/*
+  Copyright 2020 The SODA Authors.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+package com.soda.utils;
+
+/**
+ *  In this class set URL
+ */
+public class ConstantUrl {
```

Comment:
How this singleton can be used in a clustered environment?

Reply:
Clustered environment means what?

Reply:
Clustered environment means Multi-cloud Cluster with more than 1 node. Suppose s3  service is running on Node 1 & dataflow service is running on Node 2. Will the current implementation support this scenario?

Reply:
Currently we are supporting for all services on single node. We will work on this later.